gcp/appengine/blog/content/posts/2023-in-review/index.md

```diff
@@ -0,0 +1,120 @@
+it has grown to account for at least TBD% of OSV.dev's API traffic and now
+includes the following features:
+
+* [Reachability analysis](https://google.github.io/osv-scanner/experimental/#scanning-with-call-analysis) to reduce false positives
+  * [GoVulnCheck] integration to enable reachability analysis of Go vulnerabilities
+  * Experimental Rust call analysis to enable reachability analysis of Rust vulnerabilities
+* Comprehensive [C/C++ vulnerability management support](https://osv.dev/blog/posts/introducing-broad-c-c++-support/)
+  * Git submodule scanning
+  * Vendored dependency detection
+* Improved ecosystem and scanning format support
+  * Improved SBOM support
+  * NuGet
+  * ConanCenter
+* An experimental [offline mode](https://google.github.io/osv-scanner/experimental/#offline-mode)
+* An experimental [GitHub Action](https://google.github.io/osv-scanner/github-action/) (including SARIF output support)
+* An experimental license scanning feature.
+
+## More to come in 2024
+
+The team is looking forward to much more to come in 2024 and the OSV schema and
+OSV.dev’s third birthday in February, and OSV-Scanner’s first birthday in
+December. 
+
+We have a lot more exciting work planned in 2024, that we’ll share more details
+soon. Our main priorities for 2024 are centered around data quality, as well as
+providing tools for developers to help them remediate vulnerabilities faster.
+
+The main ways we plan to progress these priorities are:
+
+### 1. Validation and feedback mechanisms for OSV sources to ensure high data quality and consistency
+As OSV schema adoption grows, it’s become even more important to ensure
+consistency and high data quality across all data sources. We plan to provide
+better validation tools, and build feedback channels to  make it easier for OSV
+data sources to ensure high quality of data.
+
+### 2. Accuracy and comprehensiveness of scanning
+A continuing focus for OSV-Scanner is making sure that our scanning is
+comprehensive and accurate. Accuracy is especially important for us, as one of
+our core goals is to minimize false positives and vulnerability noise for
+developers at the receiving end of the scanners.
+
+### 3. Container scanning
+OSV-Scanner has so far focused on scanning of source repositories. One important
+gap we aim to fill is to provide better support for scanning of containers, in a
+way that provides actionable and useful remediation guidance.
+
+### 4. Remediation tools 
+Developers are often faced with an overwhelming number of vulnerabilities
+reported against their dependencies. We are currently building tooling to enable
+developers to both interactively and automatically prioritize and fix the
+vulnerabilities that matter in an easy way.
+
+Here’s a quick preview of what we have been working on: 
+
+![A sneak preview of the current UX of the guided remediation tooling under development](guided_remediation1.png "A screenshot of the guided remediation tooling under development")
```

Comment:
These two images should have slightly different alt texts and titles. Be more descriptive about what they are showing.